examples/parse-text-from-any-block-type/index.js

```diff
@@ -0,0 +1,148 @@
+import { Client } from "@notionhq/client"
+import { config } from "dotenv"
+
+config()
+
+const pageId = process.env.NOTION_PAGE_ID
+const apiKey = process.env.NOTION_API_KEY
+
+const notion = new Client({ auth: apiKey })
+
+/* 
+---------------------------------------------------------------------------
+*/
+
+// Take rich text array from a block child that supports rich text and return the plain text.
+// Note: All rich text objects include a plain_text field.
+const getPlainTextFromRichText = richText => {
+  return richText.map(t => t.plain_text).join(", ")
```

Comment:
nit: I think we want to join with just `""` vs. `", "` unless we are trying to illustrate that it's a list? 🤔

Reply:
I don't have a strong opinion on this tbh -- updating!